resolwe/flow/utils/docs/autoprocess.py

```diff
@@ -0,0 +1,395 @@
+"""
+Sphinx extension for semi-autmoatic documentation of Resolwe processes.
+
+This module introduces new directive that enables user to document a
+set of processes. The directive can be used in three variations::
+
+    .. autoprocess::
+
+    .. autoprocess:: category_name
+
+    .. autoprocess:: process_slug
+
+The first option documents *all* processes.
+
+The second one documents just the ones that have *category* equal to
+`category_name`. Since some categories have subcategories (analyses,
```

Comment:
Shouldn't you put `` `category_name` `` in double backticks?

I would change the "Since some categories ..." to "This means that subcategories (e.g. ``` ``analyses:alignment`` ```, ``` ``analyses:variants`` ```) of a category (e.g. ``` ``analyses`` ```) will *not* be documented."